.github/workflows/formatting.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
     formatting:
       name: Check Formatting
-      runs-on: ubuntu-latest
+      runs-on: ubuntu-noble
```

Comment:
Latest is now 24.04 which does not support clang-format-12. For now, just pin to older Ubuntu. WIll resolve the dependency issue long term in a formal single MR in the new year. See #446

Reply:
this is now ubuntu-22.04. Must use version number, not version name.